pandas/core/common.py

```diff
@@ -2072,7 +2072,7 @@ def _random_state(state=None):
     elif isinstance(state, np.random.RandomState):
         return state
     elif state is None:
-        return np.random.RandomState()
+        return np.random
```

Comment:
Yes. Fixed.

On 05/17, Stephan Hoyer wrote:
> > @@ -2072,7 +2072,7 @@ def _random_state(state=None):
> >      elif isinstance(state, np.random.RandomState):
> >          return state
> >      elif state is None:
> > -        return np.random.RandomState()
> > +        return np.random
> 
> can you please update the docstring, too?
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/pydata/pandas/pull/13161/files/c8a38f99637ad44dad0db9118b17fd4e3c8643f3#r63603099